tests/src/psa_exercise_key.c

```diff
@@ -778,6 +778,10 @@ int mbedtls_test_psa_exported_key_sanity_check(
             /* The representation of an ECC Montgomery public key is
              * the raw compressed point */
             TEST_EQUAL(PSA_BITS_TO_BYTES(bits), exported_length);
+        } else if(PSA_KEY_TYPE_ECC_GET_FAMILY(type) == PSA_ECC_FAMILY_TWISTED_EDWARDS) {
```

Comment:
Ah, but the CI is unhappy because we're now picky about line lengths. Please run
```
scripts/code_style.py --fix tests/src/psa_exercise_key.c

```

Reply:
Thanks